runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -136,6 +152,10 @@ private void init()
 		add(combo(5, food(2), boost(SLAYER, 4), new BoostedStatBoost(ATTACK, false, perc(.02, -3)), new BoostedStatBoost(DEFENCE, false, perc(.02, -3)), new BoostedStatBoost(STRENGTH, false, perc(.02, -3))), SLAYERS_RESPITEM, SLAYERS_RESPITEM1, SLAYERS_RESPITEM2, SLAYERS_RESPITEM3, SLAYERS_RESPITEM4);
 		add(combo(5, food(1), new SimpleStatBoost(MAGIC, true, perc(.02, 2)), new BoostedStatBoost(ATTACK, false, perc(.05, -1)), new BoostedStatBoost(DEFENCE, false, perc(.05, -1)), new BoostedStatBoost(STRENGTH, false, perc(.05, -1))), WIZARDS_MIND_BOMB, MIND_BOMB1, MIND_BOMB2, MIND_BOMB3, MIND_BOMB4);
 		add(combo(5, food(2), new SimpleStatBoost(MAGIC, true, perc(.02, 3)), new BoostedStatBoost(ATTACK, false, perc(.05, -2)), new BoostedStatBoost(DEFENCE, false, perc(.05, -2)), new BoostedStatBoost(STRENGTH, false, perc(.05, -2))), MATURE_WMB, MIND_BOMBM1, MIND_BOMBM2, MIND_BOMBM3, MIND_BOMBM4);
+		add(combo(9, food(14), boost(STRENGTH, 3), boost(MINING, 1), heal(PRAYER, perc(.06, -1)), heal(AGILITY, perc(.09, -3)), heal(ATTACK, perc(.06, -1)), heal(DEFENCE, perc(.08, -2)), heal(HERBLORE, perc(.06, -1)), heal(MAGIC, perc(.05, -1)), heal(RANGED, perc(.06, -1))), BRAINDEATH_RUM);
+		add(combo(food(2), heal(PRAYER, perc(.04, -2))), BLOODY_BRACER);
+		add(combo(2, food(1), boost(AGILITY, 1), heal(STRENGTH, -1)), ELVEN_DAWN);
+		add(combo(3, boost(RANGED, 4), heal(STRENGTH, perc(.04, -2)), heal(MAGIC, perc(.04, -2))), LIZARDKICKER);
```

Comment:
Likewise, make sure you're remembering the hp heal when calculating the `primaries` value, and use `BoostedStatBoost` for percentage skill drains here.